.github/workflows/build.yml

```diff
@@ -0,0 +1,26 @@
+name: build
+on: pull_request
+jobs:
+  frontend:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v4
+      - name: Build frontend
+        run: docker build -f ./build/package/frontend.dockerfile .
+
+  servers:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v4
+      - name: Build servers
+        run: docker build -f ./build/package/servers.dockerfile . --build-arg SERVER_TARGET=api
```

Comment:
Shouldn't we build for the other targets as well? They don't have quite the same dependencies

Reply:
I was wondering about that, but I didn't see any places where the contents are different, but indeed different things are build via the scripts. Fixed.